errors/go.mod

```diff
@@ -1 +1 @@
-module github.com/decred/dcrwallet/errors
+module github.com/decred/dcrwallet/errors
```

Comment:
You probably added an empty line here that wasn't here or you removed an empty line that was here. Please fix.